nixos/modules/services/web-servers/nginx/default.nix

```diff
@@ -397,13 +397,27 @@ in
         default = pkgs.nginxStable;
         defaultText = "pkgs.nginxStable";
         type = types.package;
+        apply = p: p.override {
+          modules = p.modules ++ cfg.additionalModules;
```

Comment:
I'd really appreciate if people ran all `nginx*` nixos tests when changing the module.
This broke the `nginx-variants` test again, because `tengine` doesn't have `.modules`. I don't use `tengine` and maybe nobody does, but the test still seems useful, because it tests some other nginx variants.

Reply:
Fixed in #119427.